src/pages/SettingsPage.tsx

```diff
@@ -19,8 +20,12 @@ const SettingsPage = () => {
         <SelectComponent className={'select-type'} options={typeOptions} text={'type'} />
       </section>
       <section className="buttons-wrapper">
-        <ButtonComponent className={'stars-btn'} text={'See my stats'} />
-        <ButtonComponent className={'start-btn'} text={'Start quiz'} />
+        <Link to={'/statistics'}>
+          <ButtonComponent className={'stars-btn'} text={'See my stats'} />
+        </Link>
```

Comment:
it can confuse other developers
use Link or Button with a handler where you'll redirect user to the corresponding page